### ISSUE TITLE
single dataset pipelines automate update of published index

### ISSUE THREAD

**Issue by tgbugs** [SciCrunch/sparc-curation#112]
need a reduced index of published dataset ids that will replace the combined curation export, it should be updated by sparcron probably, needed to simplify sds viewer

**Comment by tgbugs** [comment]
The reason these get stuck is because `hasUriPublished` is not populated without rerunning the export without fetching once publication is complete.

**Comment by jgrethe** [comment]
Probably the easiest way to get the ground truth is to pull that list from Pennsieve

**Comment by tgbugs** [comment]
Yes, adding the discover api as a third source that we check to trigger export is the full solution.
